src/connection/manager.ts

```diff
@@ -86,11 +87,14 @@ export class SQLJobManager {
     return (this.selectedJob >= 0);
   }
 
-  runSQL<T>(query: string): Promise<T[]> {
+  async runSQL<T>(query: string): Promise<T[]> {
     const selected = this.jobs[this.selectedJob]
     if (SQLJobManager.jobSupport && selected) {
-      return selected.job.query(query) as Promise<T[]>;
-
+      // 2147483647 is NOT arbitrary. On the server side, this is processed as a Java
+      // int. This is the largest number available without overflow
+      let rowsToFetch = 2147483647;
```

Comment:
Can this be a const?

Reply:
done, https://github.com/halcyon-tech/vscode-db2i/pull/23/commits/b81039529a63d4a70124bc8e7a5d945e8eda863c